### ISSUE TITLE
Unable to focus elements placed after Editor in modal Window by pressing Tab in Firefox

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#4224]
### Bug report
Tabbing out of an Editor (or iframe) nested in a modal Window, does not move the focus to the next focusable element.

### Reproduction of the problem
1. [Dojo](https://dojo.telerik.com/DWxaOeok) example.
1. Click in the Editor.
1. Press Tab => focus is not moved to the button below the Editor. 

### Current behavior
Focus is NOT moved to the button below the Editor. Instead, the Window is focused and may be moved around with arrow keys.

### Expected/desired behavior
Focus is moved to the button below the Editor as in Chrome 119.

### Environment
* **Kendo UI version:** 2023.3.1114
* **Browser:** Firefox 119.0.1

**Comment by nelito987** [comment]
This behavior was originally reported over 7 years ago and has not received significant demand or follow-up from the community since then. This is why we plan to prioritize more impactful and widely requested improvements.

If this limitation is still affecting your scenario, please share additional context or use cases. This will help us better assess the severity and reconsider prioritization in the future. Thank you!
